packages/backend/userServices.js

```diff
@@ -178,15 +178,62 @@ async function findAllReviews() {
 // function to find reviews by userId
 async function findReviewsByWrittenBy(userId) {
   try {
-    const reviews = await Reviews.find({ written_by: userId });
-    console.log("Found reviews:", reviews);
+    // also populate the album
```

Comment:
wait, so this function will find a review by user id and populate the album with album data? i think im a bit confused on why we have to populate the album here, or are we populating the reviews with albums?

Reply:
I think that's an old comment I accidently left.